mesonbuild/wrap/wrap.py

```diff
@@ -246,9 +248,19 @@ def __post_init__(self) -> None:
         self.subdir_root = os.path.join(self.source_dir, self.subdir)
         self.cachedir = os.path.join(self.subdir_root, 'packagecache')
         self.wraps = {} # type: T.Dict[str, PackageDefinition]
+        self.netrc = None
```

Comment:
Mypy objects to this, because its type changes.

```suggestion
        self.netrc: T.Optional[netrc] = None
```